reference/content-types/single_category_selection.rst

```diff
@@ -0,0 +1,60 @@
+Single Category selection
+=========================
+
+Description
+-----------
+
+Let you assign one category. Categories can be managed in the settings section of Sulu.
+The selection will be saved as a single id.
+
+Parameters
+----------
+
+.. list-table::
+    :header-rows: 1
+
+    * - Parameter
+      - Type
+      - Description
+    * - request_parameters
+      - collection
+      - Collection of parameters that are appended to the requests sent by the selection.
+    * - item_disabled_condition
+      - string
+      - This condition is evaluated foreach record in the list and is used to determine if the record is selectable.
+    * - allow_deselect_for_disabled_items
+      - boolean
+      - Defines if the user can deselect a selected item if `item_disabled_condition` is true.
+    * - form_options_to_list_options
```

Comment:
not sure if i would document this because there is no documentation about the `form_options` nor about the `list_options`. to be honest, i dont even know what `form_options` are 😂 what do you think?